pepys_import/resolvers/command_line_input.py

```diff
@@ -1,26 +1,39 @@
-def get_choice_input(heading, choices):
-    map_choice = False
-    while 1:
-        input_text = heading + "\n"
-        for idx, choice in enumerate(choices, 1):
-            if isinstance(choice, list):
-                choice_string = choice[0]
-                map_choice = True
-            else:
-                choice_string = choice
-            input_text += "   " + str(idx) + ") " + choice_string + "\n"
-        choice = input(input_text)
+from prompt_toolkit import prompt
```

Comment:
Hmm, maybe we can save this for a future issue, but...

This class was an implementation of a command-line input.  Now though, it's a wrapper for `prompt_toolkit`.

Do we need to wrap `prompt_toolkit`?  I _Guess_ we can make more advanced usage of `prompt_toolkit `if we're accessing it directly from `command_line_resolver`.

Reply:
Yes, it is a wrapper now. I tried to create the same menus with small differences (cancelling texts, title, etc.) using `create_menu` function and I think it's useful now because it prevents many duplications. However, the name of the module is confusing. 

If you would like to, I can move the function from `command_line_input` to `command_line_resolver`. Or do you have something else on your mind?